sql3/test/defs/defs_timequantum.go

```diff
@@ -99,6 +141,7 @@ var timeQuantumTest = TableTest{
 			ExpErr: "calling ranqeq() usage invalid",
 		},
 		{
+			name: "stringset-rangeq",
```

Comment:
At line 129 the expected error message has a typo. It's missing 'c' in 'calling'.